dialoptions.go

```diff
@@ -301,6 +302,17 @@ func WithPerRPCCredentials(creds credentials.PerRPCCredentials) DialOption {
 	})
 }
 
+// WithCreds returns a DialOption to set a credentials bundle for the
+// ClientConn.WithCreds. This should not be used together with
+// WithTransportCredentials.
+//
+// This API is experimental.
+func WithCreds(b credentials.Bundle) DialOption {
```

Comment:
How about changing the name to something like `WithCredentialsBundle`, or something more expressive. `WithCreds` might be a bit ambiguous about the creds type, e.g., the above API is for *transport* credentials.

Reply:
Done